src/config.rs

```diff
@@ -98,13 +99,15 @@ impl Config {
         let uda_selection_dim = Self::get_uda_selection_dim(data);
         let uda_selection_blink = Self::get_uda_selection_blink(data);
         let uda_calendar_months_per_row = Self::get_uda_months_per_row(data);
+        let uda_style_selection = Self::get_uda_style("selection", data);
```

Comment:
Can you change this to 

```suggestion
        let uda_report_style_selection = Self::get_uda_style("report.selection", data);
```

Reply:
ok